src/Components/BreathingRoom/BreathingRoom.js

```diff
@@ -1,11 +1,41 @@
 import React from 'react';
+import {Component} from 'react'
 import './BreathingRoom.scss';
 
-const BreathingRoom = () => {
+import {inhaleExhale} from './BreathingRoomAnimation.js'
```

Comment:
Good decision to have the animations be in a different file, and then just import them to the main JS component. I feel like animations can get pretty out of hand length wise pretty quickly. This will look much nicer in the long run.